### DISCUSSION TITLE
Rename File and Folder in file transfer window?

### DISCUSSION THREAD

**Discussion by luapfr** [rustdesk/rustdesk#7758] [General]
Using the file transfer option instead of the desktop makes usability difficult due to the absence of a rename option.

**Comment by rustdesk** [comment]
@21pages add this, use right click for desktop, long press for mobile.

also add tooltips to below all icons buttons

![image](https://github.com/user-attachments/assets/b7845018-afbc-425c-9e0c-df92dad174be)

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/pull/9089
